tests/test_sensor.py

```
@@ -0,0 +1,93 @@
+)
+def test_device_sensor_available(data, sensor_key, expected):
+    desc = SensorEntityDescription(key=sensor_key)
+    coordinator = DummyCoordinator(data)
+    sensor = MyAirDeviceSensor("Test Device Sensor", desc, coordinator)
+    with patch.object(MyAirDeviceSensor, "async_write_ha_state", return_value=None):
+        sensor._handle_coordinator_update()
+    assert sensor.available == expected
+
+
+@pytest.mark.parametrize(
+    "data,expected",
```

Ruff (0.12.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)